frontend-react/src/components/FileHandlers/FileHandlerForm.tsx

```diff
@@ -67,9 +78,28 @@ export const FileHandlerForm = ({
                 </div>
                 <div className="grid-col flex-1" />
                 <div className="grid-col flex-1 display-flex flex-column flex-align-end">
+                    <Dropdown
```

Comment:
Probably out of scope but would be nice for the dropdown to be hidden once the file has been validated. The user experience seems a little clunky.

Reply:
Sure, I can definitely update that!